x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/update_signals_route.test.ts

```diff
@@ -81,51 +93,56 @@ describe('update_signals', () => {
       expect(statusCode).toBe(400);
     });
 
+    test('returns 404 if the record does not exist yet', async () => {
+      alertsClient.find.mockResolvedValueOnce(getFindResult());
```

Comment:
This isn't mocked a second time/no default is set -- do you want `Once` here, or is this because you want it to not be mocked on any subsequent call?

```suggestion
      alertsClient.find.mockResolvedValue(getFindResult());
```

Reply:
You are correct. I changed it to be mockResolvedValue now